packages/wallet-ui/src/components/ui/organism/Header/SendModal/SendModal.view.tsx

```diff
@@ -64,7 +71,41 @@ export const SendModalView = ({ closeModal }: Props) => {
         break;
       case 'address':
         if (fieldValue !== '') {
-          if (!isValidAddress(fieldValue)) {
+          if (debounceRef.current) clearTimeout(debounceRef.current);
+
+          if (isValidAddress(fieldValue)) {
+            setResolvedAddress(fieldValue);
+            break;
+          } else if (isValidStarkName(fieldValue)) {
+            debounceRef.current = setTimeout(() => {
+              setLoading(true);
+              getAddrFromStarkName(fieldValue, chainId)
+                .then((address) => {
+                  if (isValidAddress(address)) {
+                    setResolvedAddress(address);
+                  } else {
+                    setResolvedAddress('');
+                    setErrors((prevErrors) => ({
+                      ...prevErrors,
+                      address: '.stark name doesn’t exist',
+                    }));
+                  }
+                })
+                .catch((error) => {
+                  if (error.name !== 'AbortError') {
+                    setResolvedAddress('');
+                    setErrors((prevErrors) => ({
+                      ...prevErrors,
+                      address: '.stark name doesn’t exist',
+                    }));
+                  }
```

Comment:
this checking can be removed?
```suggestion
     setResolvedAddress('');
       setErrors((prevErrors) => ({
        ...prevErrors,
        address: '.stark name doesn’t exist',
      }));
```